app/controllers/Giraffe.scala

```diff
@@ -84,10 +84,10 @@ class Giraffe(paymentServices: PaymentServices) extends Controller {
   val maxAmount: Option[Int] = 2000.some
 
 
-  def contributeRedirect = /*NoCache*/Action { implicit request =>
+  def contributeRedirect = NoCacheAction { implicit request =>
     val countryGroup = request.getFastlyCountry.getOrElse(CountryGroup.RestOfTheWorld)
-    val url = MakeGiraffeRedirectURL(request, countryGroup)
-    Redirect(url, SEE_OTHER)
+
+    Redirect(routes.Giraffe.contribute(countryGroup).absoluteURL(), SEE_OTHER)
```

Comment:
Ah, using `.absoluteURL()` means that the redirect goes to HTTP rather than HTTPS (because the Play app server thinks the request is insecure) - I'm going to change this to just `.url`.